golang/cosmos/x/vstorage/vstorage.go

```diff
@@ -8,22 +8,23 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 
 	"github.com/Agoric/agoric-sdk/golang/cosmos/vm"
+	"github.com/Agoric/agoric-sdk/golang/cosmos/x/vstorage/types"
 )
 
 type vstorageHandler struct {
 	keeper Keeper
 }
 
 type vstorageMessage struct {
-	Method string `json:"method"`
-	Path   string `json:"key"` // TODO: rename JSON to "path"
-	Value  string `json:"value"`
+	Method string            `json:"method"`
+	Args   []json.RawMessage `json:"args"`
```

Comment:
Is this backwards compatible? And if not, are there any external clients that will be affected?

Reply:
This is not backwards compatible, but the JS layer is updated in the same commit. There are no external clients.

Reply:
Are we sure that none of the gRPC/REST/etc. handlers end up here?

Reply:
> Are we sure that none of the gRPC/REST/etc. handlers end up here?

Yes.  This handler is only reachable by internal `cosmic-swingset` communication (currently FFI, later will be over a private pipe) between JS and Golang.